src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -270,7 +318,7 @@ public class FunctionsSystemLogsEventSource : ExtendedEventSource
         {
             internal static readonly FunctionsSystemLogsEventSource Instance = new FunctionsSystemLogsEventSource();
 
-            [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational, Version = 1)]
+            [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational, Version = 2)]
```

Comment:
I'm curious how this versioning works? Have you tried it out before?

Reply:
Yes. Version needs to be incremented any time we add columns. MDS had documentation regarding this.

Reply:
So if you deploy this with a private Site Extension today, will we see this show up in Kusto for your app?

Reply:
Yes. We can test this in Kusto!!